src/core_modules/capture-core/components/WidgetErrorAndWarning/WidgetWarning/WidgetWarning.component.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+import React from 'react';
+import { colors } from '@dhis2/ui';
+import { Widget } from '../../Widget';
+import type { widgetWarningTypes } from './WidgetWarning.types';
+import { WidgetErrorAndWarningContent } from '../content/WidgetErrorAndWarningContent';
+import { WidgetWarningHeader } from './WidgetWarningHeader';
+
+export const WidgetWarning = ({ warning }: widgetWarningTypes) => {
+    const widgetType = 'warning';
```

Comment:
I see that the type has been declare in every file, we should declare all types in a common file and reuse them. WDYT?